xAPI-Communication.md

```diff
@@ -484,15 +479,14 @@ Object.
 verify the received Statement matches the existing one and SHOULD return `409 Conflict` if they
 do not match. See [Statement comparision requirements](statement-comparision-requirements).
 
-* If the LRS receives a Statement with an id it already has a Statement for **in the same batch**, it SHOULD*
-verify the received Statement matches the existing one and SHOULD return `400 Bad Request` if they
-do not match. See [Statement comparision requirements](statement-comparision-requirements).
+* If the LRS receives a Statement with an id it already has a Statement for **in the same batch**, it SHOULD* reject the 
```

Comment:
How about: "If the LRS receives a batch of Statements containing two or more Statements with the same id..."